src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-invalid {
```

Comment:
States _should_ always be attached to a specific class, since they will be repeated frequently:
https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentnameis-stateofcomponent

Reply:
Updated that. ".is-invalid" is kind of on its own earlier in the file, so I was trying to follow the pattern there. Let me know if this works any better.